topics/ecology/tutorials/Obitools-metabarcoding/tutorial.md

```diff
@@ -0,0 +1,537 @@
+>
+> 1. {% tool [obigrep](toolshed.g2.bx.psu.edu/repos/iuc/obi_grep/obi_grep/1.2.13) %} with the following parameters:
+>    - *"Choose the sequence record selection option"*: `predicat`
+>        - *"Python boolean expression to be evaluated for each sequence record."*: `mode!="joined"`
+>
+>
+>    > ### {% icon tip %} Tip: Verifying FastQ format and converting it
+>    >
+>    > * You can use {% tool [FastQC](toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.73+galaxy0) %} to look at the format of the sequencing encoding score then {% tool [FASTQ GROOMER](toolshed.g2.bx.psu.edu/repos/devteam/fastq_groomer/fastq_groomer/1.1.5) %} to specify the guessed sequencing encoding score and create a fastqsanger file.
+>    >
+>    {: .tip}
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > The obigrep command is in some way analog to the standard Unix grep command. It selects a subset of sequence records from a sequence file.
+>    > 
+>    > A sequence record is a complex object composed of an identifier, a set of attributes (key=value), a definition, and the sequence itself.
+>    > 
+>    > Instead of working text line by text line as the standard Unix tool, selection is done sequence record by sequence record. A large set of options allows refining selection on any of the sequence record elements.
+>    > 
+>    > Moreover obigrep allows specifying simultaneously several conditions (that take the value TRUE or FALSE) and only the sequence records that fulfill all the conditions (all conditions are TRUE) are selected.
+>    > 
+>    > Sequence record selection options : 
+>    > * sequence : Regular expression pattern to be tested against the sequence itself. ex: GAATTC
+>    > * definition : Regular expression pattern to be tested against the definition of the sequence record. ex: [Cc]hloroplast
+>    > * identifier : Regular expression pattern to be tested against the identifier of the sequence record. ex: ^GH
+>    > * idlist : points to a text file containing the list of sequence record identifiers to be selected.
+>    > * attribute : Regular expression pattern matched against the attributes of the sequence record. the value of this attribute is of the form : key:regular_pattern. ex:'family_name:Asteraceae'
+>    > * hasattribute : Selects sequence records having an attribute whose key = KEY.
+>    > * predicat : Python boolean expression to be evaluated for each sequence record. The attribute keys defined for each sequence record can be used in the expression as variable names. An extra variable named ‘sequence’ refers to the sequence record itself. ex: mode!="joined"
+>    > * lmax : Keeps sequence records whose sequence length is equal or shorter than lmax. ex : 100
+>    > * lmin : Selects sequence records whose sequence length is equal or longer than lmin. ex : 100
+>    {: .comment}
```

Comment:
Perhaps this comment block could be summarized a bit and include a reference for a detailed explanation? Also, I think that it would be nice to use some of those additional options in this training, in order to explain them.

Reply:
done! Thank you!